server/controller/routes/house/getHouses.js

```diff
@@ -0,0 +1,17 @@
+const { getAllHousesData } = require('../../../database/queries/house');
+
+const getAllHouses = async (req, res, next) => {
+  let { limit, skip } = req.query;
+  if (!limit) limit = 10;
+  if (!skip) skip = 0;
```

Comment:
why do you add this condition since you make a default value on the query function?